src/main/PDF/PdfController.java

```diff
@@ -51,15 +38,27 @@ public PdfController(MongoDatabase db) {
               "ListBox",
               "TextField",
               "SignatureField"));
+
+  /*
+  REQUIRES JSON Body with:
+    - "pdfType": String giving PDF Type ("FORM", "APPLICATION", "IDENTIFICATION")
+    - "fileId": String giving id of file to be deleted
+  */
   public Handler pdfDelete =
       ctx -> {
-        String user = ctx.sessionAttribute("username");
-        String organizationName = ctx.sessionAttribute("orgName");
-        UserType privilegeLevel = ctx.sessionAttribute("privilegeLevel");
+        String user = Objects.requireNonNull(ctx.sessionAttribute("username"));
+        String organizationName = Objects.requireNonNull(ctx.sessionAttribute("orgName"));
+        UserType privilegeLevel = Objects.requireNonNull(ctx.sessionAttribute("privilegeLevel"));
+
         JSONObject req = new JSONObject(ctx.body());
-        PDFType pdfType = PDFType.createFromString(req.getString("pdfType"));
-        String fileIDStr = req.getString("fileId");
-        if (!ValidationUtils.isValidObjectId(fileIDStr) || pdfType == null) {
+        PDFType pdfType =
+            Objects.requireNonNull(PDFType.createFromString(req.getString("pdfType")));
+        String fileIDStr = Objects.requireNonNull(req.getString("fileId"));
+
+        if (pdfType == null) {
```

Comment:
We already required that pdfType is not null. This check is redundant.

Reply:
We already did, but when we do the Objects.requireNonNull, we get an internal server error on the client side. When we do this check, it will send a customer JSON error. How best should we handle this?

Reply:
i like manual checks that send descriptive messages. I think the if statement is okay for now